sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/models/CorrelationFilterImpl.java

```diff
@@ -85,11 +85,11 @@
     private String contentType;
 
     private static final class PropertiesWrapper {
-        @JacksonXmlProperty(localName = "KeyValueOfstringanyType")
+        @JacksonXmlProperty(localName = "KeyValueOfstringanyType", namespace = "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect")
```

Comment:
This code is generated so we better not change it manually.
@jianghaolu, @srnagar, @alzimmermsft does the code gen customization support this customization?

Reply:
This should be do-able using a Swagger transform to add a `namespace` sub-property to the `XML` declaration of `keyValueOfstringanyType`.

Reply:
I agree, this should not be hand-edited. Also, updating the swagger might be useful if all other languages are also doing something similar.

Reply:
This setting is added to correspond to the problem that the `namespace` attribute of the `KeyValueOfstringanyType` in the XML is a blank value. If this problem can be corrected through swagger, please inform the location of the swagger configuration file storage.

Reply:
@alzimmermsft The swagger already has namespace property. But the generated code doesn't have it.
```xml
    "KeyValue": {
      "description": "Key Values of custom properties",
      "type": "object",
      "xml": {
        "name": "KeyValueOfstringanyType",
        "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
      },
      "properties": {
        "key": {
          "type": "string",
          "xml": {
            "name": "Key",
            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
          }
        },
        "value": {
          "type": "string",
          "xml": {
            "name": "Value",
            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
          }
        }
      }
    },
```

Reply:
@srnagar, @jianghaolu, @weidongxu-microsoft, could this be an code generation issue when there is a combination of XML wrapped and namespace attribute, where the namespace attribute doesn't get propagated to the wrapped property?

Reply:
Yes, this is a codegen issue. It's missing the namespace when generating the internal wrapper classes. I'll have this fixed.

Reply:
PR to fix this - https://github.com/Azure/autorest.java/pull/1081